### DISCUSSION TITLE
【突发】黑神话demo版遭临时工泄露

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#1525] [其他]
# 原帖地址
>[https://tieba.baidu.com/p/8931099711?pid=149924241761&cid=0#149924241761](https://www.bilibili.com/bangumi/play/ss33622)
>[https://tieba.baidu.com/p/9054785942?pid=150440304037&cid=0#150440304037](https://www.bilibili.com/video/BV1PC4y1L7mq/)

**Comment by EcoFur** [comment]
应该在前面加一个“震惊”
【突发】震惊！！！黑神话demo版遭临时工泄露
